[docs] Fix statement about the order of bind: and on: (#7357)
sveltejs/svelte: site/content/docs/02-template-syntax.md

---

> If you're using `bind:` directives together with `on:` directives, their order can affect what value the bound variable has when the event handler is called.

Reviewer: This is a bit awkwardly worded in my opinion. I'd make it closer to the original:
```suggestion
If you're using `bind:` directives together with `on:` directives, the order that they're defined in affects the value of the bound variable when the event handler is called.
```
